src/integration-test/java/com/commercetools/sync/commons/utils/SphereClientUtils.java

```diff
@@ -0,0 +1,54 @@
+package com.commercetools.sync.commons.utils;
+
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.client.SphereClientConfig;
+import io.sphere.sdk.client.SphereRequest;
+import io.sphere.sdk.queries.PagedQueryResult;
+
+import javax.annotation.Nonnull;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+public class SphereClientUtils {
+
+    public static final SphereClientConfig CTP_SOURCE_CLIENT_CONFIG = SphereClientConfig.of(
+        System.getenv("SOURCE_PROJECT_KEY"),
+        System.getenv("SOURCE_CLIENT_ID"),
+        System.getenv("SOURCE_CLIENT_SECRET"));
+    public static final SphereClientConfig CTP_TARGET_CLIENT_CONFIG = SphereClientConfig.of(
+        System.getenv("TARGET_PROJECT_KEY"),
+        System.getenv("TARGET_CLIENT_ID"),
+        System.getenv("TARGET_CLIENT_SECRET"));
+    public static final SphereClient CTP_SOURCE_CLIENT =
+        ClientConfigurationUtils.createClient(CTP_SOURCE_CLIENT_CONFIG);
+    public static final SphereClient CTP_TARGET_CLIENT =
+        ClientConfigurationUtils.createClient(CTP_TARGET_CLIENT_CONFIG);
+
+    /**
+     * Max limit that can be applied to a query in CTP.
+     */
+    public static final Long QUERY_MAX_LIMIT = 500L;
+
+    /**
+     * Fetches resources of {@link T} using query provided by {@code query}.
+     * Then each resource from result list is converted to {@link SphereRequest} using {@code ctpRequest}.
```

Comment:
instead of `each resource from result list is converted to {@link SphereRequest}` . Maybe -> 
`The {@code ctpRequest} is applied on each resultant resource from fetching, to make a {@link SphereRequest}`